expressEngine/lib/responseHandler/responseHandler.js

```diff
@@ -0,0 +1,17 @@
+import { error as commonError, success } from '../../cms/common';
+
+class responseHandler {
+  success = ({ data, message }) => ({ data: data || null, message: message || success.defaultSuccessMsg });
```

Comment:
Replace
`success = ({ data, message }) => ({ data: data || null, message: message || success.defaultSuccessMsg });`
with this
`success = ({ data = null, message = success.defaultSuccessMsg }) => ({ data, message });`
Reason: Instead of adding extra checks, we can use default values while destucturing the object.